### PR TITLE
curvefs/ansible

### PR DESCRIPTION
**Pull request by Wine93**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Wine93** [comment]
recheck

File: curvefs/devops/client.yml [hunk 1]
```diff
@@ -12,5 +12,9 @@
       tags: [mount]
 
     - name: umount fs
-      shell: "fusermount3 -u {{ curvefs_client_mount_path }}"
+      shell:
+        sudo curvefs_tool umountfs
```
Comment (SeanHai) [inline curvefs/devops/client.yml:16]: need sudo?

File: curvefs/devops/roles/common/tasks/core.yml [hunk 2]
```diff
@@ -10,5 +10,12 @@
       dest_port={{ ansible_ssh_port }}
       checksum=True
     when: project_core_method == "sync_direct"
+  # TODO(@Wine93): support tarball, deb, rpm scheme
 
-    # TODO(@Wine93): support tarball, deb, rpm scheme
+  - name: install tools
```
Comment (SeanHai) [inline curvefs/devops/roles/common/tasks/core.yml:15]: Why not reuse 'sync project' ?

**Comment by Wine93** [comment]
recheck

**Comment by Wine93** [comment]
recheck

**Comment by ilixiaocui** [comment]
cc @Wine93 Please resubmit to master if necessary

**Pull request closed without merging**